jcl/src/java.base/share/classes/com/ibm/oti/util/ExternalMessages-MasterIndex.properties

```diff
@@ -1305,3 +1305,6 @@ K0900="Create a new Reference, since a Reference cannot be cloned."
 #java.lang.invoke.MethodHandle
 K0A00="Failed to resolve Constant Dynamic entry with j9class: {0}, name: {1}, descriptor: {2}, bsmData: {3}"
 K0A01="Constant_Dynamic references bootstrap method '{0}' does not have java.lang.invoke.MethodHandles.Lookup as first parameter."
+
+#java.lang.System
+K0B00="Runtime is specified to disallow setting security manager"
```

Comment:
This can be more explicit. i.e. `-Djava.security.manager=disallow` has been specified

Reply:
Updated the message.